quarkchain/tools/count_total_balance.py

```diff
@@ -14,44 +14,61 @@
 logging.getLogger("jsonrpcclient.client.response").setLevel(logging.WARNING)
 
 TIMEOUT = 10
-
-host = "http://localhost:38391"
+TOTAL_SHARD = 8
```

Comment:
Do we have a more elegant way to get the current total_shard?

Reply:
`getNetworkInfo` JSONRPC API should do, but I think for now this is fine for our network